spec/factories/form526_submissions.rb

```diff
@@ -323,4 +323,18 @@
              success: false)
     end
   end
+
+  trait :with_uploads_and_ancillary_forms do
+    form_json do
+      with_ancillary = JSON.parse(File.read("#{submissions_path}/with_everything.json"))
+      with_uploads = JSON.parse(File.read("#{submissions_path}/with_uploads.json"))['form526_uploads']
```

Comment:
Doesn't the `with_everything.json` file already include `form526_uploads`?

Reply:
No, just the ancillary forms.

Reply:
🫤 https://github.com/department-of-veterans-affairs/vets-api/blob/a882a944643e4dacd799091dc20ff66cb0b33a98/spec/support/disability_compensation_form/submissions/with_everything.json#L263

Reply:
something fails here when i try to use these 'uploads' to actually create records. it's an unrelated bug with the factory IMHO, but we can look into it in the future.